README.md

```diff
@@ -1,47 +1,23 @@
-# MongoDB Chatbot Framework
+# MongoDB Knowledge Service Framework
 
-The MongoDB Chatbot Framework is a set of libraries that you can use to build
-full-stack intelligent chatbot applications using MongoDB and [Atlas Vector Search](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/).
-The MongoDB Chatbot Framework includes first class support for
-retrieval-augmented generation (RAG).
+This repo contains the work of the MongoDB Education AI team. 
 
-The framework can take your chatbot application from prototype to production.
+## MongoDB Knowledge Service
 
-You can quickly get an AI chatbot enhanced with your data up and running using
-the framework's built-in data ingest process, chatbot server, and web UI. As you
-refine your application and scale to more users, you can modify the chatbot's
-behavior to meet your needs.
+The MongoDB Knowledge Service lets you learn about MongoDB using generative AI.
 
-The framework is flexible and customizable. It supports multiple AI models and
-complex prompting strategies. It also includes tools for programmatic evaluation of your chatbot's AI components.
+## MongoDB Chatbot Framework (deprecated)
 
-## Documentation
+The team building the MongoDB Knowledge Service previous developed the MongoDB Chatbot Framework. This consisted of the npm packages:
 
-To learn how to use the MongoDB Chatbot Framework, refer to the documentation:
-<https://mongodb.github.io/chatbot/>.
+- `mongodb-chatbot-server`
+- `mongodb-chatbot-ui` (still used, refer to [UI](./docs/ui.md))
+- `mongodb-rag-core`
+- `mongodb-rag-ingest`
 
-You can also check out the following articles and videos about the framework:
+The MongoDB Chatbot Framework in now deprecated. We will no longer be maintaining it.
```

Comment:
Maybe add a little blurb as to why?

- Third party frameworks have gotten a lot better
- Recommended alternative?: Vercel AI SDK